### DISCUSSION TITLE
pi-Base community call Thu 2024 Jan 18 3pm Eastern

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#490] [Announcements]
Agenda:
- UI change comparison: https://github.com/orgs/pi-base/discussions/495#discussioncomment-7880745
    - https://traits-table.topology.pages.dev/spaces/S000038/properties
    - https://traits-table.topology.pages.dev/properties/P000004/spaces
 - play with the navigation when entering a different branch in the Advanced tab and then clicking on links (loses the new branch sometimes, including when clicking refresh?)

Specifically, demo the following: starting on a page within a different branch (as updated at some point from the Advanced tab), if I click on a link to have it displayed in a separate browser tab, or if click on "duplicate tab", the different branch gets lost and it reverts to the main branch. This behavior happens at least on both Firefox and Microsoft Edge.

**Comment by StevenClontz** [comment]
Day/time works for me @ccaruvana and @prabau, but if you would need to meet at another time, let us know.

**Comment by StevenClontz** [comment]
I'm no longer available this week. Would someone else like to lead this discussion, or would it be okay if we push back to Jan 18?

**Reply by prabau** [reply]
Yeah, let's push it back to Jan 18.

**Reply by ccaruvana** [reply]
Yeah, that works for me! See you then.

**Comment by StevenClontz** [comment]
As discussed - will merge https://github.com/orgs/pi-base/discussions/495#discussioncomment-7880745
